doc/source/conf.py

```diff
@@ -15,9 +15,19 @@
 
 import datetime
 import subprocess
+import warnings
+
+warnings.filterwarnings(
+    "ignore",
+    message="DCDReader currently makes independent timesteps",
+    category=DeprecationWarning,
+    module="MDAnalysis.coordinates.DCD",
+)
```

Comment:
Same question as a @RMeli : What happens if you do not include this code?

I would have hoped that the only necessary change is the addition of `:okwarning:`.

According to https://ipython.readthedocs.io/en/stable/sphinxext.html you can also add `:suppress:` to silence warnings. This would be _much_ preferred to doing the filterwarnings in conf.py.

Can you please remove the whole code block here and see if you can solve the problem just with sphinx directives?